README.rst

```diff
@@ -0,0 +1,107 @@
+
+
+Monitoring-Integration
+=======================
+
+
+Functionalities
+----------------
+
+- Create a new dashboard in Grafana
+
+
+Environment Setup
+-----------------
+
+#. Clone the monitoring-integration repository
+
+    ::
+
+        $ git clone https://github.com/Tendrl/monitoring-integration.git
+	$ cd monitoring-integration
+
+
+.. note::
+
+    All the commands mentioned below are run as a regular user that has ``sudo``
+    privileges.
+    These commands are assumbed to be run from a single directory, which
+    contains the code of monitoring-integration.
+
+#. Install python pip
+
+
+   https://pip.pypa.io/en/stable/installing/
+
+
+#. Install python dependencies
+
+    ::
+
+        $ python setup.py install
+
+
+#. Install Grafana
+
+
+    http://docs.grafana.org/installation/
+
+
+#. Configure grafana for anonymous login.
+
+    ::
+
+        $ cp tendrl/monitoring_integration/grafana/grafana.ini /etc/grafana/.
```

Comment:
why you are maintaining grafana.ini in monitoring-integration and copying it to  /etc/grafana/,  Instead of that mention what is the changes in wants to do in /etc/grafana/grafana.ini